lib/helpers/cookieHandler.js

```diff
@@ -9,15 +10,40 @@ function decode(string) {
 
 //TODO make this less brittle and configurable
 module.exports = {
-  getKmtCookie: function(req, res) {
+  getKatCookie: function(req, res) {
     const cookieHandler = new Cookies(req, res, {keys:[config.get('KMT_SECRET')]});
-    const getKmtCookie = cookieHandler.get('FTkmt', {signed:true});
-    if (getKmtCookie) {
-        logger.info({operation: 'getKmtCookie', result: 'getKmtCookie cookie exists', cookieValue:getKmtCookie});
-        return decode(getKmtCookie);
+    const getKatCookie = cookieHandler.get('FTkmt', {signed:true});
+    if (getKatCookie) {
+        logger.info({operation: 'getKatCookie', result: 'getKatCookie cookie exists', cookieValue:getKatCookie});
+        return decode(getKatCookie);
     }
 
-    logger.info({operation:'getKmtCookie', result:'getKmtCookie cookie does not exist'});
+    logger.info({operation:'getKatCookie', result:'getKatCookie cookie does not exist'});
     return;
+  },
+
+  setLicenceListCookie: function(req, res, licenceList) {
+    const cookieHandler = new Cookies(req, res, {keys:[config.get('KMT_SECRET')]});
+    cookieHandler.set('FTKatL',JSON.stringify(licenceList),{
```

Comment:
@ciprianlujeru Fair comment. Cookie size is bigger when encode hence the reason I removed it but given this is  a bit of a hack anyway given I want to get the licence details somewhere other than a cookie I can see no harm in encoding it.